src/components/typography/typography.style.ts

```diff
@@ -178,7 +178,7 @@ const StyledTypography = styled.span.attrs(
     white-space: ${truncate ? "nowrap" : whiteSpace};
     word-wrap: ${wordWrap};
     text-align: ${textAlign};
-    text-overflow: ${truncate ? "ellipsis" : textOverflow};
+    text-overflow: ${truncate && !textOverflow ? "ellipsis" : textOverflow};
```

Comment:
**suggestion**: here instead could we do `text-overflow: ${textOverflow || truncate && "ellipsis"}`. Might be slightly cleaner?